kotlinpoet/src/commonMain/kotlin/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -652,6 +660,11 @@ internal class CodeWriter constructor(
       return false
     }
 
+    if (implicitModifiers.contains(KModifier.INTERNAL) || implicitModifiers.contains(KModifier.PRIVATE) || implicitModifiers.contains(KModifier.PROTECTED)) {
```

Comment:
```suggestion
    if (implicitModifiers.containsAnyOf(KModifier.INTERNAL, KModifier.PRIVATE, KModifier.PROTECTED)) {
```

(it's a handy helper defined in `Util.kt`)

Reply:
makes sense! made the change